connect/models/aniversary.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of the Ingram Micro Cloud Blue Connect SDK.
+# Copyright (c) 2019-2020 Ingram Micro. All Rights Reserved.
+
+from .base import BaseModel
+from .schemas import AniversarySchema
+
+
+class Aniversary(BaseModel):
```

Comment:
Small typo here. Should be 'Anniversary'.

Reply:
Thanks!!